src/Components/Nav/Icon.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Icon.scss';
+
+export default class Icon extends Component {
+  render() {
+    const { url } = this.props;
+    return (
+      <div>
+        <Link className="icon" to="/Signup">
+          <i class={url}></i>
```

Comment:
```suggestion
          <i className={url}></i>
```
JSX에서는 class -> className 쓰셔야 하고 콘솔에 에러 떠있을거라서 찾아서 다 고쳐주세요